dash-info.yaml

```diff
@@ -6,3 +6,782 @@ pkg_help_description: >-
+                )
+            
+                bar <- animation_opts(plot_ly(
+                  data = df, x=~time, y=~value, type = "bar"
+                  ),
+                  1000, easing = "cubic-in-out"
+                )
+                
+                return(list(bar))
+              }
+            )
+            
+            app$run_server()
+    - name: dccSlider
+      dontrun: TRUE 
+      code: |
+            library(dash)
+            library(dashCoreComponents)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(
+                list(
+                  dccSlider(
+                    id = "slider-input",
+                    min = -5,
+                    max = 10,
+                    step = 0.5,
+                    value = -3,
+                  ),
+                  htmlDiv(
+                    id = "slider-output",
+                    children = "Make a selection on the slider to see the value appear here."
+                  )
+                )
+              )
+            )
+            
+            app$callback(
+              output("slider-output", "children"),
+              list(input("slider-input", "value")),
+              function(value) {
+                return(sprintf("You have chosen %s on the slider above.", value))
+              }
+            )
+
+            app$run_server()
+    - name: dccConfirmDialog
+      dontrun: TRUE 
+      code: |
+            library(dash)
+            library(dashCoreComponents)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+            
+            app$layout(
+              htmlDiv(
+                list(
+                  dccConfirmDialog(
+                    id='confirm',
+                    message='Danger danger! Are you sure you want to continue?'
+                  ),
+            
+                  dccDropdown(
+                    options=lapply(list('Safe', 'Danger!!'),function(x){list('label'= x, 'value'= x)}),
+                    id='dropdown'
+                  ),
+                  htmlDiv(id='output-confirm1')
+                )
+              )
+            )
+            
+            app$callback(
+              output = list(id = 'confirm', property = 'displayed'),
+              params=list(input(id = 'dropdown', property = 'value')),
+              function(value){
+                if(value == 'Danger!!'){
+                  return(TRUE)}
+                else{
+                  return(FALSE)}
+              })
+            
+            
+            app$callback(
+              output = list(id = 'output-confirm1', property = 'children'),
+              params=list(input(id = 'confirm', property = 'submit_n_clicks')),
+              function(n_clicks, value) {
+                if(length(n_clicks) == FALSE){
+                  sprintf('It wasnt easy but we did it %s', str(n_clicks))
+                }
+              })
```

Comment:
It seems like this callback isn't behaving the way it should, I wasn't able to populate the children component.

Reply:
Yeah, good observation -- I think we'll chop this callback out, it's not really necessary here.